packages/api-request-builder/package.json

```
@@ -3,6 +3,10 @@
   "publishConfig": {
     "access": "public"
   },
+  "engines": {
+    "node": ">=14",
+    "npm": ">=v6.0.0"
```

I wouldn't include `npm` in the `engines` of each package, only `node`.

---

Also I think the version should be without the `v`

```suggestion
    "npm": ">=6.0.0"
```